app/views/layouts/listnav/_host.html.haml

```diff
@@ -77,6 +77,12 @@
               ems_infra_path(@record.ext_management_system),
               :title => _("Show this parent %{provider} for this %{host}") % {:host => host_title, :provider => ui_lookup(:table => "ems_infra")})
 
+        - if role_allows?(:feature => "physical_server_show") && !@record.physical_server.nil?
+          %li
+            = link_to("#{ui_lookup({:table => "physical_servers"})}: #{@record.physical_server.name}",
```

Comment:
Currently the text is being shown as plural `Physical Servers`... we need it to be singular.
I have created https://github.com/ManageIQ/manageiq/pull/15275 with which `ui_lookup` should work correctly and show the singular text.